ansible/playbooks/molecule/teleop_server_check_desktop_icons_docker/tests/test_teleop_server.py

```diff
@@ -0,0 +1,35 @@
+import os
+import testinfra.utils.ansible_runner
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('all')
+
+
+def test_icons_in_docker(host):
+    user = host.user().name
+    icon_list = []
+    script_list = []
+    desktop_path = '/home/'+str(user)+'/Desktop/'
```

Comment:
please add spaces before and after + sign

Reply:
done